CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Added the `MapboxNavigationService.router(_:didPassVisualInstructionPoint:routeProgress:)` and `MapboxNavigationService.router(_:didPassSpokenInstructionPoint:routeProgress:)` methods, which correspond to `Notification.Name.routeControllerDidPassVisualInstructionPoint` and `Notification.Name.routeControllerDidPassSpokenInstructionPoint`, respectively. ([#1912](https://github.com/mapbox/mapbox-navigation-ios/pull/1912))
 * Added an initializer to `DispatchTimer`, along with methods for arming and disarming the timer. ([#1912](https://github.com/mapbox/mapbox-navigation-ios/pull/1912))
 
+### Navigation
+* Renamed `CarPlayManagerDelegate.carPlayManager(_:mapButtonsCompatibleWith:in:for:)` to `CarPlayManagerDelegate.carPlayManager(_:mapButtonsCompatibleWith:in:for:carPlayMapViewController:)`. You can now create and return map buttons that perform built-in actions on the map by calling methods such as `CarPlayMapViewController.zoomInButton()`. ([#1962](https://github.com/mapbox/mapbox-navigation-ios/pull/1962))
+
```

Comment:
@1ec5 Ideally, it makes more sense to create the map buttons using a `lazy computed property`. Lazy computed properties are only calculated once.